debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
> name it debian/$pkg.service

That's exactly what is happening here. There is debian/mgmt.service file which is being automatically installed.